server/serverHandlerFunc_test.go

```diff
@@ -0,0 +1,110 @@
+package server
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestServerGetFunc(t *testing.T) {
+	var testServer = NewServer()
+	err := testServer.GetFunc("/test", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	})
+
+	if err != nil {
+		t.Errorf("Error Occurred: %v", err)
+	}
+
+	request, _ := http.NewRequest(http.MethodGet, "/test", nil)
+	response := httptest.NewRecorder()
+
+	testServer.ServeHTTP(response, request)
+
+	if response.Code != http.StatusOK {
+		t.Fatalf("Something wrong, code : %v", response.Code)
+	}
+}
+
+func TestServerPutFunc(t *testing.T) {
+	var testServer = NewServer()
+	err := testServer.PutFunc("/test", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	})
+
+	if err != nil {
+		t.Errorf("Error Occurred: %v", err)
+	}
+
+	request, _ := http.NewRequest(http.MethodPut, "/test", nil)
+	response := httptest.NewRecorder()
+
+	testServer.ServeHTTP(response, request)
+
+	if response.Code != http.StatusOK {
+		t.Fatalf("Something wrong")
+	}
+}
+
+func TestServerPutFuncHandlerNotFound(t *testing.T) {
+	var testServer = NewServer()
+	err := testServer.PutFunc("/test", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [string literal "/test" appears, at least, 5 times, create a named constant for it](https://app.codacy.com/gh/hrple/common/pullRequest?prid=5576416)